frontend/package.json

```diff
@@ -33,14 +32,15 @@
         "react-bootstrap": "^1.4.0",
         "react-bootstrap-typeahead": "^5.1.4",
         "react-copy-to-clipboard": "^5.0.3",
-        "react-custom-scrollbars": "^4.2.1",
         "react-dom": "^17.0.1",
         "react-icons": "^4.1.0",
         "react-notification-system-redux": "^2.0.1",
         "react-redux": "^7.2.2",
         "react-router-dom": "^5.2.0",
         "react-scripts": "4.0.3",
+        "react-scrollbars-custom": "^4.0.25",
```

Comment:
Why 4.0 and not 4.2?

Reply:
This is a new library. The original library was not well maintained (last updated 2017) and it did not have enough functionality to achieve our goal. One of the main obstacles was that the old library, when creating the div for the scroll bars, does not give them a className, thus making it impossible to override their default styles or aligning them with the table. This library's latest version is [4.0.25](https://www.npmjs.com/package/react-scrollbars-custom), which includes functionalities like verticle/horizontal only, flexible scrollbar width and properly tagged div containers.

Reply:
Ah. I didn't notice that it had a different name!